doc/source/whatsnew/v2.0.0.rst

```diff
@@ -37,13 +37,15 @@ The ``use_nullable_dtypes`` keyword argument has been expanded to the following
 
 * :func:`read_csv`
 * :func:`read_excel`
+* :func:`read_html`
 * :func:`read_sql`
 
 Additionally a new global configuration, ``mode.nullable_backend`` can now be used in conjunction with the parameter ``use_nullable_dtypes=True`` in the following functions
 to select the nullable dtypes implementation.
 
 * :func:`read_csv` (with ``engine="pyarrow"``)
```

Comment:
cc @mroeschke: This seems to work for engine=python as well, can we remove the engine restriction?

Reply:
Oh really? Does it work with the "c" engine as well? Otherwise we can just clarify it's just for the python and pyarrow engines

Reply:
Good point, no only python engine. Will open another pr to adjust whatsnew and add a test